tests/std/tests/P0896R4_ranges_alg_remove/test.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <ranges>
+#include <span>
+#include <utility>
+
+#include <range_algorithm_support.hpp>
+using namespace std;
+using P = pair<int, int>;
+
+// Validate dangling story
+STATIC_ASSERT(same_as<decltype(ranges::remove(borrowed<false>{}, 42)), ranges::dangling>);
+STATIC_ASSERT(same_as<decltype(ranges::remove(borrowed<true>{}, 42)), ranges::subrange<int*>>);
+
+struct instantiator {
+    static constexpr P expected[3] = {{0, 99}, {2, 99}, {4, 99}};
+
+    template <ranges::forward_range Read>
+    static constexpr void call() {
+        using ranges::remove, ranges::subrange, ranges::equal, ranges::iterator_t;
+
+        size_t comparisonsCounter = 0;
+        auto projection           = [&comparisonsCounter](const P& data) {

```

Comment:
Naming nitpick: Given `using namespace std`, I think that the parameter name `data` should be avoided due to shadowing. Occurs in other files.

Reply:
changed to `val`